java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -141,7 +146,8 @@ public static Object[] transactionTestResult() {
             2L,
             OK,
             Map.of("timeout", "1000"),
-            OK
+            OK,
+            1L, // pfadd(hllKey1, new String[] {"a", "b", "c"})
```

Comment:
I agree that having this comment helps indicate which command is associated with this result, but it looks a bit strange having a comment for this one specific result but none of the others. Should decide between having comments for all of them or having comments for none of them?